### DISCUSSION TITLE
How to accept multiple types for a field?

### DISCUSSION THREAD

**Discussion by drewg2009** [rebing/graphql-laravel#1011] [Q&A]
I want to be able to accept multiple types for a field in my GraphQL Type that's returned from my query. 

For example right now I have a field like this                 

            'type_data' => [
                'type' => Type::string(),
                'description' => 'json string of type data returned by applicable models based on the type'
            ],

But I would like to use types instead of json so I don't have to decode it on the frontend. Is something like this possible? 

            'type_data' => [
                'type' => Type::Industry | Type::Tier | Type::Vertical
                'description' => 'data for the type of model that is referenced'
            ],

**Comment by mfn** [comment]
If I understand you correctly, you want to look into unions -> https://github.com/rebing/graphql-laravel#unions

**Reply by drewg2009** [reply]
Yes but I want to use union types inside the fields array for a given field, not the return type for an entire query
